### PR TITLE
[develop/fetch] Add nodes to adjust the volume of the speaking

### PR DESCRIPTION
**Pull request by Kanazawanaoaki**
`develop/fetch` version of https://github.com/jsk-ros-pkg/jsk_robot/pull/1715
This PR adds Apps to adjust the volume of Fetch `audible_warning` and `tweet`.

### CODE REVIEW

**Comment by mqcmd196** [comment]
Could you push only commits for Fetch? It seems it includes PR2's ones

**Comment by Kanazawanaoaki** [comment]
Understood.
I have removed the changes not related to fetch.

File: jsk_fetch_robot/jsk_fetch_startup/apps/volume_zero/volume_zero.app [hunk 1]
```diff
@@ -0,0 +1,6 @@
+display: Fetch Volume Zero
+description: Fetch speaks zero volume
+platform: fetch
```
Comment (knorth55) [inline jsk_fetch_robot/jsk_fetch_startup/apps/volume_zero/volume_zero.app:3]: we can use `platform: all`.
with that, we can move the app to `jsk_robot_startup` and use the same app for `fetch`, `pr2` and other robotcs.

**Review by knorth55 — CHANGES_REQUESTED** [review]

**Comment by knorth55** [comment]
I also want to make an icon.
You can make an icon from free images.

**Comment by tkmtnt7000** [comment]
I'm not sure, but I have a little concern that it may be difficult to merge to master branch with icon made from いらすとや because of some policies...

**Comment by knorth55** [comment]
how about these ?
https://pixabay.com/images/search/mute%20sound/?manual_search=1

**Comment by Kanazawanaoaki** [comment]
Thanks for the review and comments.  
As I mentioned in https://github.com/jsk-ros-pkg/jsk_robot/pull/1715#issuecomment-1334875994, I also thought it would be a good idea to put the app in `jsk_robot_startup` and create icons, so I'll work on it!

**Comment by Kanazawanaoaki** [comment]
The icon image has been rebased as follows. This PR was rebased to resolve conflicts and tested with fetch15.  
| Volume lower | Volume reset | Volume zero |
| -- | -- | -- |
| volume_lower | volume_reset | volume_zero |

**Review by knorth55 — APPROVED** [review]

**Review by mqcmd196 — APPROVED** [review]
LGTM

**Pull request merged**